#75: Upgrade to Metabase v.0.48.0 (#76)
exasol/metabase-driver: doc/developer_guide/developer_guide.md

1. Checkout Metabase at `$HOME/git/metabase` (= `$METABASE_DIR`) and build it:

    ```sh
    cd $HOME/git
    git clone https://github.com/metabase/metabase.git
    cd metabase
    git fetch --all --tags
    export METABASE_VERSION=v0.48.0
    git reset --hard
    rm -vf target/patch_excluded_test_applied
    git checkout "tags/${METABASE_VERSION}" -b "${METABASE_VERSION}-branch"
    # Build (this will take ~15min)
    ./bin/build.sh
    ```

2. Download the Exasol JDBC driver from the [Download Portal](https://downloads.exasol.com/clients-and-drivers) and install it:

    ```sh
    cp exajdbc.jar "$METABASE_DIR/plugins"
    ```

3. Checkout the Exasol Metabase driver at `$HOME/git/metabase` (= `$METABASE_EXASOL_DRIVER`)

    ```sh
    git clone https://github.com/exasol/metabase-driver.git
    cd metabase-driver
    ```

## Upgrading Metabase

> To ensure compatibility we need to regularly update the the latest Metabase version. You can find the latest Metabase version on the [GitHub release page](https://github.com/metabase/metabase/releases/).

Reviewer: ```suggestion
To ensure compatibility we need to regularly update to the latest Metabase version. You can find the latest Metabase version on the [GitHub release page](https://github.com/metabase/metabase/releases/).
```

Metabase publishes two variants:
* OSS: version numbers v0.x.y
* Enterprise: version numbers v1.x.y

We only use the OSS variant with version numbers v0.x.y.

To upgrade Metabase follow these steps:
1. Check the Metabase [driver changelog](https://www.metabase.com/docs/latest/developers-guide/driver-changelog.html) for breaking changes
2. Replace the previous version in all files with the new version by searching for `v0.x.y`
3. [Run unit tests](#running-driver-unit-tests)
4. [Run integration tests](#running-integration-tests)

The following things can go wrong:

> * Patch with excluded test fails to apply. See [excluded tests](#excluded-tests) for details.

Reviewer: ```suggestion
* The patch excluding inapproriate tests cannot be applied. See [excluded tests](#excluded-tests) for details.
```
